akvo/rest/views/project.py

```diff
@@ -151,6 +152,13 @@ def get_queryset(self):
             ).distinct()
         return super(ProjectIatiExportViewSet, self).get_queryset()
 
+    def list(self, request, *args, **kwargs):
+        projects = self.queryset.filter(run_iati_checks=True)
+        projects.update(run_iati_checks=False)
```

Comment:
Do this after the update_iati_check or as part of it? Just in case there is some exception.

Reply:
Yes, moved it to be inside `update_iati_checks`. Thanks for the catch. I was trying to not miss out on checks that are "scheduled" while this loop is running. But, moving the change inside the loop should be good enough.